apps/web/components/ui/UsernameAvailability/index.tsx

```diff
@@ -40,32 +59,32 @@ export const UsernameAvailabilityField = ({
     },
   });
 
-  const UsernameAvailability = getUsernameAvailabilityComponent(!IS_SELF_HOSTED && !user.organization?.id);
```

Comment:
It's bad practice to define a component inside another component. It causes many problems, especially unnecessary re-rendering. That is happening here